Project-Aurora/Project-Aurora/Settings/Layers/EqualizerLayerHandler.cs

```diff
@@ -120,9 +132,10 @@ public override void Default()
             _EQType = EqualizerType.PowerBars;
             _ViewType = EqualizerPresentationType.SolidColor;
             _MaxAmplitude = 20.0f;
-            _DimBackgroundOnSound = false;
+            _ScaleWithSystemVolume = false;
+            _BackgroundMode = EqualizerBackgroundMode.Disabled;
             _DimColor = Color.FromArgb(169, 0, 0, 0);
-            _Frequencies = new SortedSet<float>() { 60, 170, 310, 600, 1000, 2000, 3000, 4000, 5000 };
+            _Frequencies = new SortedSet<float>() { 50, 95, 130, 180, 250, 350, 500, 620, 700, 850, 1200, 1600, 2200, 3000, 4100, 5600, 7700, 10000, 10001, 10002 };
```

Comment:
The last few values (10k+) are my personal preference to move the other bars a bit to the left. I think the last 2 frequencies should be removed from the default ones.